mne_bids/write.py

```diff
@@ -1157,6 +1190,10 @@ def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
             recording date will be overwritten as well. If True, keep subject
             information apart from the recording date.
 
+        ``keep_source`` : bool
+            If ``True`` (default), the filename of the raw source fill will
+            be stored in the sidecar JSON ``Sources`` key.
```

Comment:
Is `Sources` upper- or lowercase?

From the other code I had the impression that we're writing to both, TSV and JSON sidecars. The description here does not seem to match this?

Reply:
Sorry it should be lower-case as it's stored in the scans.tsv file only. Not the JSON.